### DISCUSSION TITLE
Welcome to NATSpeech Discussions!

### DISCUSSION THREAD

**Discussion by RayeRen** [NATSpeech/NATSpeech#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vikas1440** [comment]
python data_gen/tts/runs/preprocess.py --config $CONFIG_NAME

I am getting

File "data_gen/tts/runs/preprocess.py", line 1, in
import utils.commons.single_thread_env # NOQA
ModuleNotFoundError: No module named 'utils.commons'

while running the above command I have installed the utils package still getting the same issue please help me resove this issue

**Reply by theabraham2000** [reply]
@RayeRen I am also facing the same issue. It would be helpful if you could give a video tutorial or a blog explaining how to run the inference for different text.

When I ran this in GColab: ! CUDA_VISIBLE_DEVICES=0 python tasks/run.py --exp_name ps_normal_exp --infer

I am getting this error

Traceback (most recent call last):
  File "/content/NATSpeech/tasks/run.py", line 5, in <module>
    from utils.commons.hparams import hparams, set_hparams
ModuleNotFoundError: No module named 'utils.commons'
